sourceCode.py

```diff
@@ -1 +1,150 @@
+                    Output 'Tell me more.'", style="filled", fillcolor="orange", shape = 'box')
+node_d2 = pydot.Node("Maintain Conversation \n\
+                    Output 'Tell me more.'", style="filled", fillcolor="orange", shape = 'box')
+# example: Open questions and fillers in that class of that specific example topic - Tell me more! How did that happen? I understand. 
+node_e = pydot.Node("End Conversation \n\
+                    Output 'Thank you for trusting me. Do you want to pick another topic or do you want to say goodbye?'", style="filled", fillcolor="yellow", shape = 'box')
+node_e1 = pydot.Node("End Conversation \n\
+                    Output 'Thank you for trusting me. Do you want to pick another topic or do you want to say goodbye?'", style="filled", fillcolor="yellow", shape = 'box')
+node_e2 = pydot.Node("End Conversation \n\
+                    Output 'Thank you for trusting me. Do you want to pick another topic or do you want to say goodbye?'", style="filled", fillcolor="yellow", shape = 'box')
+# example: Ok, you wanted to end this conversation. Do you want to choose another topic?
+
+
+# creating nodes for participants response
+
+# add nodes
+graph.add_node(node_a)
+graph.add_node(node_b)
+graph.add_node(node_c)
+graph.add_node(node_c1)
+graph.add_node(node_c2)
+graph.add_node(node_d)
+graph.add_node(node_d1)
+graph.add_node(node_d2)
+graph.add_node(node_e)
+graph.add_node(node_e1)
+graph.add_node(node_e2)
+
+
+
+# create edges
+graph.add_edge(pydot.Edge(node_a, node_b))
+graph.add_edge(pydot.Edge(node_b, node_c))
+graph.add_edge(pydot.Edge(node_b, node_c1))
+graph.add_edge(pydot.Edge(node_b, node_c2))
+
+graph.add_edge(pydot.Edge(node_c, node_d))
+graph.add_edge(pydot.Edge(node_c, node_e))
+graph.add_edge(pydot.Edge(node_c1, node_d1))
+graph.add_edge(pydot.Edge(node_c1, node_e1))
+graph.add_edge(pydot.Edge(node_c2, node_d2))
+graph.add_edge(pydot.Edge(node_c2, node_e2))
+graph.add_edge(pydot.Edge(node_e, node_b, label="alternatively go back to picking a topic", labelfontcolor="#009933", fontsize="10.0", color="blue"))
+
+graph.write_png('C:/Users/User/Documents/file2.png')
+
+## make a Jordan function
+
+class Jordan
+    def Jordan(input):
+
+
+# initiate introduction
+
+from psychopy.event import waitKeys
+
+def intro(word):
+    word = 'ENTER'
+    word.waitKeys(5)
+    return 'Welcome. I am Jordan. What is your name?'
+
+# open questions library
+
+openQuestions = [
+    []]
+
+# curse words library
+
+
+
+# open answers library 
+
+openAnswers = [
+    []]
+
+# from first to second person translation library
+
+2ndPersonPerspective = {
+  "am"   : "are",
+  "was"  : "were",
+  "i"    : "you",
+  "i'd"  : "you would",
+  "i've"  : "you have",
+  "i'll"  : "you will",
+  "my"  : "your",
+  "are"  : "am",
+  "you've": "I have",
+  "you'll": "I will",
+  "your"  : "my",
+  "yours"  : "mine",
+  "you"  : "me",
+  "me"  : "you"
+}
+
+
+## tokenize: part sentence into its single words
```

Comment:
Just wondering how many words you want to include and on which basis you choose them?
Also I just remembered something that might help you with categorising: there's a program called LIWC that codes words and analyses them, mostly for linguistic research. While I don't think they're open source, they do have a PDF indicating how they categorise which words. It does not consist of a complete word list but rather shows what categorises they are. You can find it here: https://repositories.lib.utexas.edu/bitstream/handle/2152/31333/LIWC2015_LanguageManual.pdf

Reply:
I agree - that seems like a good idea! Perhaps you could also restrict the number of words you want to include and rather stay more conservative. Especially for the scope of this project, it seems more manageable!